Examples/BookingSystem.AspNetFramework/Stores/SessionStore.cs

```diff
@@ -246,9 +246,27 @@ protected override void GetOrderItems(List<OrderItemContext<SessionOpportunity>>
                                                  x.OrderTable.ProposalStatus != ProposalStatus.SellerRejected &&
                                                  x.OccurrenceId == occurrences.Id &&
                                                  x.OrderId != flowContext.OrderId.uuid)
-                                     }
+                                     },
+                                     Attendee = flowContext.Customer != null
+                                         ? new Person
+                                         {
+                                             GivenName = flowContext.Customer.GivenName,
+                                             FamilyName = flowContext.Customer.FamilyName,
+                                             Email = flowContext.Customer.Email,
+                                             Telephone = flowContext.Customer.Telephone
+                                         }
+                                         : null,
```

Comment:
Don't need this - it should be set by the test suite on the request, not on the response
```suggestion
```